hazelcast-client-new/src/main/resources/hazelcast-client-config-3.5.xsd

```diff
@@ -401,10 +401,10 @@
         </xs:sequence>
     </xs:complexType>
     <xs:complexType name="ssl">
-        <xs:sequence>
+        <xs:all>
```

Comment:
@pveentjer no,  `all` means 0 or 1 for every item. So user cannot configure `size` more than 1.